packages/terra-search-field/tests/nightwatch/search-field-spec.js

```diff
@@ -3,19 +3,19 @@ const resizeTo = require('terra-toolkit/lib/nightwatch/responsive-helpers').resi
 
 module.exports = resizeTo(['tiny', 'small', 'medium', 'large', 'huge', 'enormous'], {
   'Displays a search field with search button': (browser) => {
-    browser
-      .url(`${browser.launchUrl}/#/tests/search-field-tests/default`)
-      .assert.elementPresent('#searchfield')
-      .assert.elementPresent('#searchfield button')
-      .assert.attributeEquals('#searchfield input', 'placeholder', '');
+    browser.url(`${browser.launchUrl}/#/tests/search-field-tests/default`);
+
+    browser.expect.element('#searchfield').to.be.present.to.equal(true);
```

Comment:
Doesn't `to.be.present` already assert if the element exists in the dom? `to.equal(true)` seems redundant

Reply:
it is needed to be done to get around the no-unused-expressions linter rule, else I would need to turn that off